### PR TITLE
feat(pathway to success) : new page (pathway-to-success) created.

### PR DESCRIPTION
**Pull request by yashcrest**
This PR creates UI for `Pathway to success`  as per the requirements in issue #69 #73 #74 .

@Aaasisss , @20was could you please review and provide feedback. 

## Release Notes

- **New Features**
	- Introduced the `HeroSection` component to promote the "Pathway to Success" theme.
	- Launched the `CareerSuccessCard` component for showcasing detailed career success information, including titles and key topics.
	- Enhanced the `CareerSuccessCard` with a responsive design and a "Learn More" button for user engagement.
	- Added a new data structure for career-related insights, enriching the content available to users.

These enhancements aim to improve user engagement and provide valuable resources for career development.

### CODE REVIEW

File: src/data/careerSuccessData.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+export type CareerData = {
```
Comment (thorn-pod) [inline src/data/careerSuccessData.ts:1]: Either make type `CareerSuccessData` or change the constant to `careerData` just for consistency.

File: src/modules/pathway-to-success/CareerSuccessGuide/Card/CareerSuccessCard.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+  CardFooter,
+} from "../../../../components/ui/card";
```
Comment (thorn-pod) [inline src/modules/pathway-to-success/CareerSuccessGuide/Card/CareerSuccessCard.tsx:8]: Can we use @ import please ? This is not bad but not good too.

**Comment by thorn-pod** [comment]
@yashcrest Please check ai comments too. Small fix required. Rest all good. Also , can you post the screenshot of the UI please ? would be easier to examine.

**Comment by yashcrest** [comment]
@20was here you go dai.

image


image


Mobile view: 

https://github.com/user-attachments/assets/b36543ed-7ef6-4975-8c37-2b06fab0bc2a

**Comment by thorn-pod** [comment]
@yashcrest Just fix the build issue. Rest looks good.

**Review by thorn-pod — APPROVED** [review]

**Review by thorn-pod — APPROVED** [review]

**Review by pratibhapaudel — APPROVED** [review]

**Pull request merged**